src/functionalTest/resources/features/F-009/S-185_CreationDataForRoleAssignment.td.json

```diff
@@ -0,0 +1,58 @@
+{
+  "_guid_": "S-185_CreationDataForRoleAssignment",
+  "_extends_": "CreationDataBaseForRoleAssignment",
```

Comment:
As this new scenario is using `CreationDataBaseForRoleAssignment` rather than `CreationDataForRoleAssignment` like some of the others: can you override `request.body.roleRequest.reference` with ```"reference": "${[scenarioContext][customValues][generateUUID]}"``` so the new test scenario uses a generated GUID rather than a hardcoded one that is also invalid.

Reply:
thanks for the feedback. addressed in commit [55719bd](https://github.com/hmcts/am-role-assignment-service/pull/2359/commits/55719bd8079e16910891584a0e3f3e56c1b8ddbb)